base/threadingconstructs.jl

```diff
@@ -110,15 +112,48 @@ A barrier is placed at the end of the loop which waits for all tasks to finish
 execution.
 
 The `schedule` argument can be used to request a particular scheduling policy.
-The only currently supported value is `:static`, which creates one task per thread
-and divides the iterations equally among them. Specifying `:static` is an error
-if used from inside another `@threads` loop or from a thread other than 1.
+Options are:
+- `:static` is the default schedule which creates one task per thread and divides the
```

Comment:
Actually, the default is pretty close to `:static` but it falls back to the "single-thread mode" on the non-primary threads. That's why it's already OK to use `@threads for` in arbitrary tasks even though the performance may not be great.

Reply:
Ah I see. Would this be more accurate?
```
The `schedule` argument can be used to request a particular scheduling policy.
Options are:
- `:static` which creates one task per thread and divides the iterations equally among
            them, assigning each task specifically to each thread.
            Specifying `:static` is an error if used from inside another `@threads` loop
            or from a thread other than 1.
- `:dynamic` which is like `:static` except the tasks are assigned to threads dynamically,
            allowing more flexible scheduling if other tasks are active on other threads.
            Specifying `:dynamic` is allowed from inside another `@threads` loop and from
            threads other than 1.

When no schedule is specified, if called from thread 1 the default is `:static` and from 
other threads the loop will be executed without threading.
```